Gemfile

```diff
@@ -39,7 +39,8 @@ gem 'jemalloc'
 gem 'customerio'
 
 group :development, :test do
-  gem 'travis-migrations', git: 'https://github.com/travis-ci/travis-migrations'
+  gem 'travis-migrations', git: 'https://github.com/travis-ci/travis-migrations', branch: 'igor-logs-migrations'
+  gem 'sequel'
```

Comment:
Can we make the Travis-migrations gem have the sequel dependency so we don't have to require it again here (and anywhere we use Travis-migrations)